notebooks/03.09-Pivot-Tables.ipynb

```diff
@@ -80,9 +80,9 @@
        "      <th>sex</th>\n",
```

Comment:
<p>This is <strong>bold</strong> comment.</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Reconsider this,

```c= a+b```

- A
- B
- C

Reply:
<p><code>d = a +b </code></p><p><br></p><ul><li>A</li><li>B</li><li>C</li></ul>